src/lib/token_list_gen.ts

```diff
@@ -128,9 +132,28 @@ export const generateTokenList = async (
     }).sort((a, b) => (a.symbol < b.symbol ? -1 : 1))
     console.log(`List has ${unbridgedL1Tokens.length} unbridged tokens`);
     
+  }
+
+  if(options && options.includeAllL1Tokens) {
+    l2TokenList = l2TokenList.concat(l1TokenList.tokens.filter(
+        (l1TokenInfo) => l1TokenInfo.chainId !== parseInt(l2Network.chainID)
+      )
+      .map((l1TokenInfo)=>{
+        return {
+          chainId: +l1TokenInfo.chainId,
+          name: l1TokenInfo.name,
+          address: l1TokenInfo.address,
+          symbol: l1TokenInfo.symbol,
+          decimals: l1TokenInfo.decimals,
+          logoURI: l1TokenInfo.logoURI
+        }
+    }).sort((a, b) => (a.symbol < b.symbol ? -1 : 1)))
```

Comment:
Why do we sort this here? Any reason not to keep the original order?

Reply:
the order is important here if you want to be able to easily identify where the arbified part of the list starts.
just appending to the original might work well

Reply:
Removed the sort